### PR TITLE
Add dark theme.

### PR DESCRIPTION
**Pull request by knightburton**
Add a global dark style theme to the project.
This can be changed in the settings menu, under the theme option.
Note: Before start using the iotjscode with this, the browser
locale storage should be cleared because the stored structure is modified.

IoT.jsCode-DCO-1.0-Signed-off-by: Imre Kiss kissi.szeged@partner.samsung.com

### CODE REVIEW

File: src/app/view/workspace/debugger-buttons.ejs [hunk 1]
```diff
@@ -7,7 +7,7 @@
         <i class="fa fa-plug" aria-hidden="true"></i> Connect to</div>
       <input type="text" name="host-ip" id="host-ip" class="form-control input-sm" placeholder="address" title="The host ip address." value="localhost">
       <span>:</span>
-      <input type="number" name="host-port" id="host-port" class="form-control input-sm" placeholder="port" title="The host port number." min=0 max=65535 value="5001">
+      <input type="text" name="host-port" id="host-port" class="form-control input-sm" placeholder="port" title="The host port number." value="5001">
```
Comment (daeyeon) [inline src/app/view/workspace/debugger-buttons.ejs]: Can't `type="number min=0 max=65535"` remain?

Comment (knightburton) [reply src/app/view/workspace/debugger-buttons.ejs]: Sure!

File: src/app/app.scss [hunk 2]
```diff
@@ -991,3 +1007,291 @@ input[type="number"] {
...
+  textarea,
+  select {
+    border: none;
+    color: #000 !important;
+    box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
+    -moz-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
+    background: #dfdfdfb2;
+  }
+
+  input[type=number]::-webkit-inner-spin-button {
```
Comment (daeyeon) [inline src/app/app.scss]: IMO, it doesn't seem as important as having to use non-standards. How about removing this style?

Comment (knightburton) [reply src/app/app.scss]: Well, you are right, I will remove this.

**Pull request merged**